### ISSUE TITLE
Selective data deletion?

### ISSUE THREAD

**Issue by do-at** [afliw/chromium-cleaner#1]
I found it when I was looking for a way to selectively delete local data upon exiting Chromium.  I think this extension works really well for users who want everything gone on exit! Thank you for writing it! 

Seeing that it deletes more than just the browsing history (cookies, site data, etc. is all being deleted by the extension as well), would it be possible to add an option menu or something where users can choose exactly what they want deleted?

**Comment by afliw** [comment]
Yes, it's possible. I don't have much time right now to implement it, but whenever I can, I will add the menu.
